### PR TITLE
fixing another path error since there's sub folders now

### PR DESCRIPTION
**Pull request by mbreakey3**
This change is [Reviewable](https://reviewable.io/reviews/powershell/dscresource.tests/63)

### CODE REVIEW

File: DscResourceCommonTests/Meta.Tests.ps1 [hunk 1]
```diff
@@ -26,7 +26,7 @@ Import-Module -Name (Join-Path -Path $TestHelper -ChildPath 'TestHelper.psm1') -
 $ErrorActionPreference = 'stop'
 Set-StrictMode -Version latest
 
-$RepoRoot = (Resolve-Path $PSScriptRoot\..).Path
+$RepoRoot = Split-Path -Parent (Split-Path -Parent $PSScriptRoot.Path)
```
Comment (KarolKaczmarek) [inline DscResourceCommonTests/Meta.Tests.ps1]: could you update me why we are making this change?
Also, I think we should use $Parent to get repoRoot

**Comment by kwirkykat** [comment]
@mbreakey3 Are you ready for this change yet?

**Comment by mbreakey3** [comment]
I think this ties into what our new change will be for running the tests

**Comment by kwirkykat** [comment]
@mbreakey3 Oh. This isn't going into dev. I see.

**Comment by kwirkykat** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[DscResourceCommonTests/Meta.Tests.ps1, line 29 \[r1\]](https://reviewable.io:443/reviews/powershell/dscresource.tests/63#-KJDTa1xuS9VPxotLHL5-r1-29:-KOHnNeThxuQ8xRMS-LX:360568521) ([raw file](https://github.com/powershell/dscresource.tests/blob/09d8fff8d700321eaa91bb74de56d90a4b78e6f5/DscResourceCommonTests/Meta.Tests.ps1#L29)):*
<details><summary><i>Previously, KarolKaczmarek (Karol Kaczmarek) wrote…</i></summary>
> could you update me why we are making this change?
> 
> Also, I think we should use $Parent to get repoRoot
</details>
Mariah - $Parent is already defined above this line ^ You should either replace that variable or change the  $script:MyInvocation... part to $PSScriptRoot and then just use that variable instead.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/63)*

**Comment by mbreakey3** [comment]
Closing until we come up with better solution for the larger issue of running pester tests

**Pull request closed without merging**